.circleci/config.yml

```diff
@@ -98,6 +98,42 @@ jobs:
               echo "Skip the Artman Docker image publishing step."
             fi
 
+  release:
+    working_directory: /usr/src/artman
+    docker:
+      - image: docker:17.05.0-ce-git
+    steps:
+      - *checkout_artman
+      - setup_remote_docker
+      - *restore_artman_cache
+      - *load_build_and_save_artman_image
+      - *save_artman_cache
+      - run:
+          name: Decrypt PyPI configuration file
+          command: |
+            openssl aes-256-cbc -d -in .circleci/.pypirc.enc -out ~/.pypirc -k "${PYPIRC_ENCRYPTION_KEY}"
+      - run:
+          name: Push Artman Docker tagged image
+          command: |
+            if [ "${DOCKER_EMAIL}" == 'googleapis-publisher@google.com' ]; then
+              # extract version number from setup.py
+              ARTMAN_VERSION=`awk -F"'" '/version=/ { print $2; }' setup.py`
+              # release to PyPI
+              echo "Releasing artman version ${ARTMAN_VERSION} to PyPI."
+              python3 setup.py sdist upload
+              # push to docker
+              docker login -e "${DOCKER_EMAIL}" -u "${DOCKER_USER}" -p "${DOCKER_PASS}"
+              docker tag "${IMAGE_HASH}" "googleapis/artman:${ARTMAN_VERSION}"
+              docker push "googleapis/artman:${ARTMAN_VERSION}"
```

Comment:
Don't we need to tag & push `googleapis/artman:latest`?

Reply:
We do it in another step but I agree it's better to do it here with the same image we just built. Fixing